src/SuperDump.Analyzer.Linux/Program.cs

```diff
@@ -2,28 +2,55 @@
 
 using SuperDump.Analyzer.Linux.Boundary;
 using SuperDump.Analyzer.Linux.Analysis;
+using System.Collections.Generic;
 
 namespace SuperDump.Analyzer.Linux {
 	public class Program {
+		private const string EXPECTED_COMMAND = "CoreDumpAnalysis [-prepare] coredump-file|coredump-dir [output-file]";
```

Comment:
you could thing about using `CommandLineParser` library for doing commandline stuff. It's already used in https://github.com/Dynatrace/superdump/blob/master/src/SuperDump.DebugDiag/Program.cs